test/utils/mockutils/mockcond/conditions.go

```diff
@@ -0,0 +1,61 @@
+package mockcond
```

Comment:
For the sake of simplicity, could we just have mockserver have both the server and conditions?

Reply:
It will conflate the meaning. `mockcond` should autocomplete all possible checks one can have on response body. While `mockserver` is about building, setting up the server. There are few options when doing autocomplete, easier to search for methods, no need to remember which goes where.

Relation:
mockserver uses mockcond